src/installer.js

```diff
@@ -127,6 +127,7 @@ class DebianInstaller extends common.ElectronInstaller {
 
     this.defaults = Object.assign(common.getDefaultsFromPackageJSON(pkg), {
       version: transformVersion(pkg.version || '0.0.0'),
+      revision: undefined,
```

Comment:
This is the default in `-common`:

https://github.com/electron-userland/electron-installer-common/blob/5bab86dbb8db9e7cec5298fef17bd02529a7c842/src/defaults.js#L29

```suggestion
```